src/Coordination/KeeperSnapshotManager.cpp

```diff
@@ -210,11 +211,11 @@ void KeeperStorageSnapshot<Storage>::serialize(const KeeperStorageSnapshot<Stora
         writeBinary(snapshot.zxid, out);
         if (keeper_context->digestEnabled())
         {
-            writeBinary(static_cast<uint8_t>(Storage::CURRENT_DIGEST_VERSION), out);
+            writeBinary(static_cast<uint8_t>(KEEPER_CURRENT_DIGEST_VERSION), out);
```

Comment:
Minor question: Why do we need all the casting to `uint8_t` if the class is declared as `KeeperDigestVersion : uint8_t`?